recipe/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+$PYTHON setup.py install --conda
+
+cp ${RECIPE_DIR}/xonsh_shortcut.json ${PREFIX}/Menu/xonsh_shortcut.json
+cp ${RECIPE_DIR}/xonsh.ico ${PREFIX}/Menu/xonsh.ico
```

Comment:
Seems that the linux package for spyder has a diffent folder structure:
share/
   applications/
    - spyder3.desktop
   pixmaps
    - spyder3

Hmm. All of this is probably an undocumented conda feature. Maybe something Continuum uses to put application launchers in the packages they build them self. Maybe @asmeurer  knows something about this. 

Anyway. I will remove the Menu entry. I don't see a way to only apply it on the Windows, since we use conda convert to build the packages.